elements/lisk-validator/src/validation.ts

```diff
@@ -190,18 +190,19 @@ export const validatePublicKeys = (
 	publicKeys.every(validatePublicKey) &&
 	validatePublicKeysForDuplicates(publicKeys);
 
-const ADDRESS_BYTE_LENGTH = 20;
-export const validateAddress = (address: string): boolean => {
-	if (!isHexString(address)) {
-		throw new Error('Address is not in hex format');
-	}
+const ADDRESS_LENGTH = 40;
```

Comment:
address length must be 41?

Reply:
`41` is for base32 address. this is `binaryAddress`